lib/view/view_team_info.dart

```diff
@@ -0,0 +1,228 @@
+                          child: PopupMenuButton<String>(
+                            color: AppColor.tertiaryColor,
+                            itemBuilder: (BuildContext context) =>
+                                <PopupMenuEntry<String>>[
+                              PopupMenuItem<String>(
+                                value: 'Option 1',
+                                child: Container(
+                                  // decoration: const BoxDecoration(
+                                  //   border: Border(
+                                  //     bottom: BorderSide(
+                                  //       color: AppColor.accentColor,
+                                  //       width: 1,
+                                  //     ),
+                                  //   ),
+                                  // ),
+                                  child: const Text(
+                                    'The Bitter Rangers',
+                                    style: TextStyle(color: Colors.white),
+                                  ),
+                                ),
+                              ),
+                              PopupMenuItem<String>(
+                                value: 'Option 2',
+                                child: Container(
+                                  // decoration: const BoxDecoration(
+                                  //   border: Border(
+                                  //     bottom: BorderSide(
+                                  //       color: AppColor.accentColor,
+                                  //       width: 1,
+                                  //     ),
+                                  //   ),
+                                  // ),
+                                  child: const Text(
+                                    "CPI's Hurricane",
+                                    style: TextStyle(color: Colors.white),
+                                  ),
+                                ),
+                              ),
+                              PopupMenuItem<String>(
+                                value: 'Option 3',
+                                child: Container(
+                                  // decoration: const BoxDecoration(
+                                  //   border: Border(
+                                  //     bottom: BorderSide(
+                                  //       color: AppColor.accentColor,
+                                  //       width: 1,
+                                  //     ),
+                                  //   ),
+                                  // ),
+                                  child: const Text(
+                                    'None at None',
+                                    style: TextStyle(color: Colors.white),
+                                  ),
+                                ),
+                              ),
+                            ],
```

Comment:
https://api.flutter.dev/flutter/material/DropdownMenu/width.html
e
https://stackoverflow.com/questions/48895195/how-can-we-change-the-width-padding-of-a-flutter-dropdownmenuitem-in-a-dropdown